pandas/core/arrays/categorical.py

```diff
@@ -1659,7 +1659,9 @@ def _validate_codes_for_dtype(cls, codes, *, dtype: CategoricalDtype) -> np.ndar
     # -------------------------------------------------------------
 
     @ravel_compat
-    def __array__(self, dtype: NpDtype | None = None) -> np.ndarray:
+    def __array__(
+        self, dtype: NpDtype | None = None, copy: bool | None = None
```

Comment:
Could you add `copy` under `Parameters` in the docstring?

`/home/runner/work/pandas/pandas/pandas/core/arrays/categorical.py:1662:PR01:pandas.Categorical.__array__:Parameters {'copy'} not documented`